README.md

```diff
@@ -1,3 +1,9 @@
+<!--
+SPDX-FileCopyrightText: 2024 3mdeb <contact@3mdeb.com>
+
+SPDX-License-Identifier: MIT
```

Comment:
Where does the MIT came from @Al-an-21 ? We clearly already ship Apache license in this repo: https://github.com/Dasharo/open-source-firmware-validation/blob/develop/LICENSE